server/block/ladder.go

```diff
@@ -86,17 +86,17 @@ func (l Ladder) EncodeItem() (name string, meta int16) {
 
 // EncodeBlock ...
 func (l Ladder) EncodeBlock() (string, map[string]any) {
-	return "minecraft:ladder", map[string]any{"facing_direction": int32(l.Facing + 2)}
+	return "minecraft:ladder", map[string]any{"facing_direction": int32(l.Facing)}
 }
 
 // Model ...
 func (l Ladder) Model() world.BlockModel {
-	return model.Ladder{Facing: l.Facing}
+	return model.Ladder{Facing: cube.Direction(l.Facing)}
 }
 
 // allLadders ...
 func allLadders() (b []world.Block) {
-	for i := cube.Direction(0); i <= 3; i++ {
+	for i := cube.Face(0); i <= 5; i++ {
```

Comment:
@TwistedAsylumMC shall I use cube.Faces() instead of this?

Reply:
Yep